packages/eslint-config-udemy-website/index.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
             files: '**/*.spec.js',
             rules: {
                 'udemy/no-non-zero-settimeout': ['error', 'always'],
+                'udemy/no-then': ['error', 'always'],
```

Comment:
Remember to follow the Conventual Commits standard for your commit message. This is a non-breaking change.